modular_zubbers/code/game/objects/items/robot/items/storage.dm

```diff
@@ -228,6 +228,11 @@
 					/obj/item/survivalcapsule/,
 					/obj/item/extraction_pack,
 					/obj/item/fulton_core,
+					/obj/item/gun/energy/recharge/kinetic_accelerator,  // SPLURT EDIT. In case if your fellow miners forgets carrying...
+					/obj/item/borg/upgrade/modkit,						// ...their things with them or they dropped them while dead.
+					/obj/item/kinetic_crusher,							// Who's a good boy~?
+					/obj/item/skeleton_key,								// Yes! You are!
+					/obj/item/reagent_containers/hypospray/medipen/survival  // Also a SPLURT edit.
```

Comment:
you can modularize the addition of these the same way vendor stuff is added

Reply:
I've tried to take a KA during the testmerge and the weapon's sprite just became micro-sized 😹😹😹😹